tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_container_validation.py

```diff
@@ -372,6 +372,18 @@ def test_valid_containers(
             raw_bytes="ef00010100040100010400000000800000fe",
             validity_error=EOFException.MISSING_CODE_HEADER,
         ),
+        Container(
+            # EOF code containing multiple type headers followed by 2 code sections
+            name="multiple_type_headers_3",
+            sections=[
+                Section(kind=SectionKind.TYPE, data="00800000"),
+                Section(kind=SectionKind.TYPE, data="00800000"),
+                Section.Code(Op.INVALID),
```

Comment:
It could be that this section JUMPFs into the second section so validation could not fail due to unreferenced code sections.

Reply:
Done.